<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="mainBackground">
   <widget class="QLabel" name="BackgroundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background: transparent;
	border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="BackGround.qrc">:/Background/Background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbCopyRight">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>321</width>
      <height>15</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(255, 170, 0);
    background: transparent;
	border: none;
}</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string notr="true">CopyRight@ by Tuan</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::RichText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextInteractionFlag::NoTextInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbMyIP">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(255, 170, 0);
    background: transparent;
	border: none;
}</string>
    </property>
    <property name="text">
     <string>255.255.255.255</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>20</y>
      <width>293</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="RingMeter" name="meCPU" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbCpuInfo">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 9pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
         </property>
         <property name="text">
          <string>Load Avg: ----</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="RingMeter" name="meRam" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbRamInfo">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>9</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 9pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
         </property>
         <property name="text">
          <string>RAM: ---/---MB</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>150</y>
      <width>293</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lbTemperPicture">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    background: transparent;
	border: none;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="temper.qrc">:/images/temper.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>CPU</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbCpuTemp">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>*C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>RAM</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbRamTemp">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>*C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>GPU</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbGpuTemp">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>--</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 6pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
             </property>
             <property name="text">
              <string>*C</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="VeticalLine">
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 0);</string>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lbCurrentTime">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: rgb(85, 255, 255);
	font: 700 20pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
         </property>
         <property name="text">
          <string>--:--:--</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="margin">
          <number>-5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbCurrentDate">
         <property name="maximumSize">
          <size>
           <width>16777214</width>
           <height>10</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 9pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
         </property>
         <property name="text">
          <string>DD/MM/YYY</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="margin">
          <number>-5</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>Up Time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbUpTime">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>15</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: rgb(255, 255, 255);
	font: 8pt &quot;Segoe UI&quot;;
    background: transparent;
	border: none;
}</string>
           </property>
           <property name="text">
            <string>--:--:--</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>261</width>
      <height>2</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>2</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 0);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <zorder>BackgroundLabel</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>lbCopyRight</zorder>
   <zorder>lbMyIP</zorder>
   <zorder>line</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RingMeter</class>
   <extends>QWidget</extends>
   <header>RingMeter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="BackGround.qrc"/>
  <include location="temper.qrc"/>
 </resources>
 <connections/>
</ui>
